scripts/test.backend.sh

```diff
@@ -15,6 +16,16 @@ else
   cargo build --locked --target wasm32-unknown-unknown --release -p signer
 fi
 
+if [ -f "./ic-btc-canister.wasm.gz" ]; then
+  # Setting the environment variable will be used in the test to load that particular file. Relative to where the test is.
+  echo "Use existing ic-btc-canister.wasm.gz canister."
+  export BITCOIN_CANISTER_WASM_PATH="../../ic-btc-canister.wasm.gz"
```

Comment:
Just for info:

In one of my projects I had to move a crate, so that it was nested one deeper.  All the tests that used `../../` to find .wasm files broke.  So I now use code like this:
```
   /// Gets the CARGO_WORKSPACE directory, which is typically the root of the git repository.
    fn workspace_dir() -> PathBuf {
        let output = std::process::Command::new(env!("CARGO"))
            .arg("locate-project")
            .arg("--workspace")
            .arg("--message-format=plain")
            .output()
            .unwrap()
            .stdout;
        let cargo_path = Path::new(std::str::from_utf8(&output).unwrap().trim());
        cargo_path.parent().unwrap().to_path_buf()
    }
    /// The path to a canister Wasm after `dfx deploy`.  Expects the Wasm to be gzipped.
    ///
    /// If not already gzipped, please add this to the canister declaration in `dfx.json`:
    /// ```
    ///    "gzip": true`
    /// ```
    fn dfx_wasm_path(name: &str) -> String {
        Self::workspace_dir()
            .join(format!(".dfx/local/canisters/{name}/{name}.wasm.gz"))
            .to_str()
            .unwrap()
            .to_string()
    }
```

I don't think you need to switch the Wasm paths, certainly not in this PR, but you might like to be aware of the option. :-)

Reply:
I stressed a bit (also in my own project) about  having a nice clean way of getting Wasms without having shell scripts all over the place.  The best I could think of, although I wasn't very happy with it, was to:

* Deploy the canister locally (with `dfx deploy $canister`)
* Get the Wasm from `.dfx/local/canisters/$canister/$canister.wasm.gz`
  (hence that `dfx_wasm_path()` function above, that I use in pocket-ic tests)

This works regardless of how the canister is defined.  A remote Wasm that is downloaded or a local one that is built, it is the same.  But this involves actually building the Wasm and I wish that were not necessary.  Still, in CI I can test that `dfx deploy` works out of the box, then use all the assets running that created for other tests.